cmake/tribits/ci_support/filter-packages-list.py

```diff
@@ -43,17 +43,20 @@
 
 
 #
-# Read in the commandline arguments
+# Read in the command-line arguments
 #
 
 usageHelp = \
 r"""filter-packages-list.py --deps-xml-file=<PROJECT_DEPS_FILE> \
   --input-packages-list=<P1>,<P2>,... --keep-test-test-categories=<T1>,<T2>,...
 
-This script takes in a comma-seprated list of TriBITS package names
-<P1>,<P2>,... and then filters out the package names for packages that don't
-the package test test categories listed in <T1>,<T2>,... which are given the
-TriBITS-generated project dependencies file.
+This script takes in a comma-separated list of TriBITS package names in
+--input-packages-list='<P1>,<P2>,...' and then filters out the names for
+packages that don't match the set categories listed in
+--keep-test-test-categories='<T1>,<T2>,...' (where each package's test test
```

Comment:
"test test" (that's OK though, fix later :-) )

Reply:
@mhoemmen, "test test categories" is actually not a typo.  See:

* https://tribits.org/doc/TribitsDevelopersGuide.html#test-test-category